src/app/app.component.css

```diff
@@ -0,0 +1,59 @@
+mat-toolbar-row {
+  background: #0070C0
+}
+
+mat-sidenav-container {
+  height: 100%;
+}
+
+mat-sidenav {
+  width: 200px;
+  height: 1000px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background: #0070C0;
+}
+
+mat-selection-list {
+  color: white;
+}
+
+.side {
+  background: #0389c7;
+}
+
+.navText {
+  color: white;
+}
+
+.pet-card {
+  padding-top: 50px;
+  min-width: 75%;
+}
+
+.pet-field {
+  color: #0070C0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#0070C0" to be "#0070c0" (color-hex-case)](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES-My-Pet-Care-Web/pullRequest?prid=5648599)